src/libraries/System.Reflection.Metadata/tests/PortableExecutable/PEReaderTests.cs

```diff
@@ -849,5 +851,45 @@ public void KeepMetadataAlive()
 
             Assert.Equal(@"Debug", reader.GetString(reader.GetAssemblyDefinition().Name));
         }
+
+        [Fact]
```

Comment:
Failing on .netframework test run
```suggestion
        [Fact]
        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
```